app/renderer/js/pages/preference/find-accounts.js

```diff
@@ -0,0 +1,83 @@
+'use-strict';
+
+const request = require('request');
+
+const BaseComponent = require(__dirname + '/../../components/base.js');
+
+class FindAccounts extends BaseComponent {
+	constructor(props) {
+		super();
+		this.props = props;
+	}
+
+	template() {
+		return `
+			<div class="settings-card find-accounts-card">
+				<div class="email-input">
+					<div>Email address(es)</div>
+					<input class="setting-input-value" autofocus placeholder="Add up to 10 comma-separated email addresses"/>
+				</div>
+				<div class="email-input">
+					<button class="green sea w-150" id="find-accounts-button">Find accounts</button>
+				</div>
+			</div>
+		`;
+	}
+
+	confirmation() {
```

Comment:
How about de-duplicating confirmation and error function here, since they used same html code but diffrent messages into a new function that return the html with message like:

```javascript
	getMessageElement(msg) {
		return `
			<div class="sub-title">
				<div>${msg}</div>
			</div>
		`;
	}

	confirmation() {
		const confirmationMsg = 'Emails sent! You will only receive ' +
		'emails at addresses associated with Zulip organizations.'; 
	  	return this.getMessageElement(confirmationMsg);
	}
	
	error() {
		const errorMsg = 'Sorry, looks like something went wrong. ' +
		'Try again?';
		return this.getMessageElement(errorMsg);
	}
```